__tests__/app.test.js

```diff
@@ -39,4 +40,29 @@ describe('GET/api/topics', () => {
             expect(error.msg).toBe("Cannot find path")
         })
     });
+});
+
+describe('GET/api', () => {
+    test('Should return a 200 status code', () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+    });
+    test('Should return an object describing all available endpoints', () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then((response) => {
+            expect(response.body).toEqual(endpoints)
+        })
+    });
+    test('should return appropriate error when endpoint is mis-typed', () => {
+        return request(app)
+        .get("/ap")
+        .expect(404)
+        .then((response) => {
+            const error = response.body
+            expect(error.msg).toBe("Cannot find path")
```

Comment:
Good that you've thought about this, but you don't have to test a 'wrong endpoint' 404 more than once (could this possibly have failed?). NB: You will however still need to test for non-existent article ID and the like